platform/config/service/config/user_k8s_platform-bootstrap.cfg

```diff
@@ -0,0 +1,190 @@
+#
+# NOTE: IF THIS FILE IS BEING MODIFIED, please look at platform.py start() to check if
+# the ordering guarantees are broken!
+#
+version: "v1"
+name: "Applatix Platform Config Standard"
+spec:
+  steps:
+    # Step 0: Create all kubernetes namespaces
+    - name: "Essential Kubernetes Namespaces"
+      policy: "CreateOnce"
+      consistency: "CreateIfNotExist"
+      objects:
+        - name: "axuser-namespace"
+          file: "axuser-namespace.yml.in"
+
+        - name: "axsys-namespace"
+          file: "axsys-namespace.yml.in"
+
+    # Step 1: Any kubernetes addons.
```

Comment:
step 1 is needed because some other code expects it to be?

Reply:
Yes. platform.py requires it this way.